internal/provisioner/backup_restore_operator_test.go

```diff
@@ -461,7 +461,7 @@ func TestOperator_TriggerRestore(t *testing.T) {
 
 func setJobActiveWhenExists(t *testing.T, client v1.JobInterface, name string) {
 	ctx := context.Background()
-	err := wait.Poll(1*time.Second, 30*time.Second, func() (bool, error) {
+	err := wait.PollUntilContextTimeout(ctx, time.Second, 5*time.Second, true, func(ctx context.Context) (bool, error) {
```

Comment:
Have you set this timeout to 5s intentionally? It was 30s.

Reply:
Nice catch :)